Bottom: 1b074ac0f7b32825d8447f106bffe21194c9ce16
Top:    54c20b84519429afbf61c963d0cd7aed0872625f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-21 13:38:11 +0000

Refresh of butcher-the-xor-crosshair-draw

---

diff --git a/src/thindraw.c b/src/outline_draw.c
similarity index 100%
rename from src/thindraw.c
rename to src/outline_draw.c
